openslides/agenda/migrations/0003_auto_20170818_1202.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-18 10:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
So, whats your proposal? Didn't we do this previously to create new permissions?